e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -70,9 +70,24 @@ private void assertThatAccessStructuredWasFetched(String conversationId, Documen
         assertThat(accessStructured.get("taskId")).isNotNull();
     }
 
-    private void assertThatAccessDocumentWasFetched(String documentId, List<Document> accessDocuments) {
-        Document document = accessDocuments.get(0);
-        assertThat(document.get("objectName")).isEqualTo(documentId + ".json");
+    private Document theDocumentTaskUpdatesTheRecord(String conversationId) {
+        var gpcAccessDocument = (Document) Mongo.findEhrExtractStatus(conversationId).get(GPC_ACCESS_DOCUMENT);
+        return getFirstDocumentIfItHasObjectNameOrElseNull(gpcAccessDocument);
+    }
+
+    private Document getFirstDocumentIfItHasObjectNameOrElseNull(Document gpcAccessDocument) {
+        var documentList = gpcAccessDocument.get("documents", Collections.emptyList());
```

Comment:
Could an empty list default value actually break a test? We would still have a functioning list despite it being empty.

Reply:
empty list default won't break this

Reply:
Ok